lib/logic/models/pull-request.js

```diff
@@ -137,4 +149,24 @@ PullRequest.prototype.getActionsToDo = function() {
     });
 };
 
+PullRequest.prototype.isLGTM = function() {
+    if (!this.getComments().length) {
+        return false;
+    }
+    var lastComment = this.getComments()[this.getComments().length - 1];
+    return lastComment.body.indexOf("LGTM") >= 0;
+};
+
+PullRequest.prototype.getLastCommenter = function() {
+    if (!this.getComments().length) {
+        return undefined;
```

Comment:
Please return `null`.

The rationale here is that JS errors in general tend to be due to bad accesses resulting in `undefined`. By returning `null`, you more clearly eloquate the difference between a "whoopsie I typoed a var name" and "no really, this is an expected no value".